src/Centreon/Domain/Repository/NagiosServerRepository.php

```diff
@@ -51,7 +53,82 @@ class NagiosServerRepository extends ServiceEntityRepository
      */
     public function checkListOfIds(array $ids): bool
     {
-        return $this->checkListOfIdsTrait($ids, NagiosServer::TABLE, NagiosServer::ENTITY_IDENTIFICATOR_COLUMN);
+        return $this->checkListOfIdsTrait($ids);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getPaginationList($filters = null, int $limit = null, int $offset = null, $ordering = []): array
```

Comment:
```suggestion
    public function getPaginationList($filters = [], int $limit = null, int $offset = null, $ordering = []): array
```

Reply:
the method has to be the same as in the interface 
 see code here https://github.com/centreon/centreon/blob/master/src/Centreon/Infrastructure/CentreonLegacyDB/Interfaces/PaginationRepositoryInterface.php#L51